packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -585,13 +586,16 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
     /**
      * Closes the connection and clears inbound & outbound queues.
      */
-    public close(error?: ICriticalContainerError): void {
-        if (this.closed) {
+     public close(error?: ICriticalContainerError, emitDisposed?: boolean): void {
```

Comment:
This still confused me but I think I get it.  Two thoughts:
1. Calling this "andDispose" or "doDispose" or something more comprehensive would help
2. Should the "closed" event always fire even if you directly call dispose?  Would that simplify or complicate things?

And please update the doc comment for this function explaining what that parameter does with a brief reminder of close v. dispose semantics (for me the important thing to get is you can close then dispose but not dispose then close)